pytest_jira.py

```diff
@@ -274,6 +301,29 @@ def pytest_addoption(parser):
                     default=_get_value(config, 'DEFAULT', 'version'),
                     help='Used version'
                     )
+    group.addoption('--jira-marker-strategy',
+                    action='store',
+                    dest='jira_marker_strategy',
+                    default=_get_value(config, 'DEFAULT', 'strategy', 'open'),
+                    help='''Action if issue ID was not found
+                    open - issue is considered as open (default)
+                    strict - raise an exception
+                    ignore - issue id is ignored
+                    warn - write error message and ignore
```

Comment:
I think you can use 'choices' option to restrict values

https://docs.python.org/dev/library/argparse.html#choices